src/framework/MOM_diag_mediator.F90

```diff
@@ -395,6 +274,10 @@ subroutine set_axes_info(G, GV, param_file, diag_cs, set_vertical)
   call define_axes_group(diag_cs, (/ id_xh, id_yq /), diag_cs%axesCv1, &
        x_cell_method='mean', y_cell_method='point', is_h_point=.false.)
 
+  do i=1, size(diag_cs%diag_remaps)
+    call diag_remap_set_vertical_axes(diag_cs%diag_remaps(i), G, GV, param_file)
+  enddo
```

Comment:
When I came to extending this code for the additional vertical axes this code blew up a lot in size. i.e. multiply the above by <number of vertical axes> * 2 (interfaces and layers). So I switched to using the axes handles. I'd be interested to hear if you have any thoughts about this. It would be easier if I could find a way to continue to use the axes type.